shell/initialize/client.js

```diff
@@ -274,7 +272,6 @@ async function render(to, from, next) {
   // Update ._data and other properties if hot reloaded
   Components.forEach((Component) => {
     if (Component._Ctor && Component._Ctor.options) {
-      Component.options.asyncData = Component._Ctor.options.asyncData;
       Component.options.fetch = Component._Ctor.options.fetch;
     }
   });
```

Comment:
Based on your other comment about HMR[^1], would we be able to drop this as well?

[^1]: https://github.com/rancher/dashboard/pull/10896/files#r1583518001

Reply:
We might be able to but I haven't verified yet. It wasn't as obvious while I was digging. I'll take a look and open a separate PR if we can.